### PR TITLE
docs: Change Microsoft to AKS & ACA and introduce name for managed support options

### PR DESCRIPTION
**Pull request by tomkerkhove**
Change Microsoft to AKS & ACA and introduce name for managed support options

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

File: layouts/shortcodes/support.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <a href="{{ .url }}" target="_blank">
       <img src="{{ $logo }}">
     </a>
+    <p class="is-centered">{{ .name }}</p>
```
Comment (tomkerkhove) [inline layouts/shortcodes/support.html:9]: @thisisobate I suck at styling, would you mind helping me better format the name? If it's too annoying for you, we can do it ina separate PR as well.

**Review by zroubalik — APPROVED** [review]

File: layouts/shortcodes/support.html (hunk 1 above)
Comment (tomkerkhove) [reply layouts/shortcodes/support.html:9]: @hisisobate Would you have some time to help on the styling here?

**Review by JorTurFer — APPROVED** [review]

File: layouts/shortcodes/support.html (hunk 1 above)
Comment (thisisobate) [reply layouts/shortcodes/support.html:9]: Hi...sorry for the late response. For some weird reason, I couldn't get notified by GitHub about your comment.

Would take a look now and see how best to do te styling.

[the remark this answers is not in the document]
Comment (thisisobate) [reply layouts/shortcodes/support.html:9]: Append this style to `assets/sass/style.sass`:

```css
@media screen and (min-width: 769px)
       .md-height-desktop
            height: 9rem
```

**Review by thisisobate — CHANGES_REQUESTED** [review]

**Comment by tomkerkhove** [comment]
Will fix DCO

**Review by JorTurFer — APPROVED** [review]

**Review by thisisobate — APPROVED** [review]
LGTM!

**Comment by tomkerkhove** [comment]
Thanks for the help @thisisobate!
